core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -7413,10 +7413,10 @@ components:
         - $ref: "#/components/schemas/LedgerTransaction"
         - type: object
           required:
-            - flash_transaction
+            - flashed_state_updates
```

Comment:
_(note from author)_

A flash transaction, apart from state updates, also has a `name`.
Do we want to include it in the Core API response?

Reply:
Yes please :)

Reply:
(already done - see `- name` one line above)